### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by HideakiImamura**
## Contributor Agreements

Please read the [contributor agreements](https://github.com/optuna/optunahub-registry/blob/main/CONTRIBUTING.md#contributor-agreements) and if you agree, please click the checkbox below.

- [x] I agree to the contributor agreements.

## Motivation

Add issue template. This is almost the same as those of optuna/optuna.

### CODE REVIEW

**Comment by HideakiImamura** [comment]
@y0z Could you review this PR?

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      description: |
+        Please give us your environment information. You can get this information by typing the following.
+        ```
+        python3 -c 'import optuna; print(f"- Optuna version:{optuna.__version__}")'
```
Comment (y0z) [inline .github/ISSUE_TEMPLATE/bug-report.yml:23]: It is also better to ask about the version of optunahub (`optunahub.__version__`).

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        python3 -c 'import platform; print(f"- Python version:{platform.python_version()}")'
+        python3 -c 'import platform; print(f"- OS:{platform.platform()}")'
+        ```
+      value: |
+        - The package of OptunaHub which you are using:
+        - Optuna version:
```
Comment (y0z) [inline .github/ISSUE_TEMPLATE/bug-report.yml:29]: Add
- optunahub version:

**Review by y0z — COMMENTED** [review]
I leave minor comments.

**Comment by HideakiImamura** [comment]
@y0z Thanks for the review. I applied all of your suggestions. PTAL.

**Review by y0z — APPROVED** [review]
LGTM

**Pull request merged**